--- Makefile	2012-03-01 12:22:50.000000000 +0000
+++ Makefile	2013-08-29 19:39:43.370287894 +0100
@@ -12,13 +12,14 @@
 
 ##### Build defaults #####
 LUA_VERSION =       5.1
+LUA_JIT =           luajit-2.0
 TARGET =            cjson.so
-PREFIX =            /usr/local
+PREFIX =            ../../install/zproxy
 #CFLAGS =            -g -Wall -pedantic -fno-inline
 CFLAGS =            -O3 -Wall -pedantic -DNDEBUG
 CJSON_CFLAGS =      -fpic
 CJSON_LDFLAGS =     -shared
-LUA_INCLUDE_DIR =   $(PREFIX)/include
+LUA_INCLUDE_DIR =   $(PREFIX)/include/$(LUA_JIT)
 LUA_CMODULE_DIR =   $(PREFIX)/lib/lua/$(LUA_VERSION)
 LUA_MODULE_DIR =    $(PREFIX)/share/lua/$(LUA_VERSION)
 LUA_BIN_DIR =       $(PREFIX)/bin
